packages/components/src/select-control/stories/index.tsx

```diff
@@ -23,6 +23,7 @@ const meta: ComponentMeta< typeof SelectControl > = {
 		prefix: { control: { type: 'text' } },
 		suffix: { control: { type: 'text' } },
 		value: { control: { type: null } },
+		onChange: { action: 'onChange' },
```

Comment:
An alternative that we could look into adopting across the package is to use a regex to mark as actions all `on*` props (see for example [`FocalPointPicker`'s Storybook parameters](https://github.com/WordPress/gutenberg/blob/4efdb495b2ce535a03fb4765797fd0af1b5c9ec8/packages/components/src/focal-point-picker/stories/index.tsx))

Reply:
Thanks. Didn't know you could do that. Would be great to get `onBlur` and `onFocus` as well.  I'll update the PR later today.

Reply:
Updated the PR to match all `on*` using regex.